operator/integration-tests/model-in-image/mii-sample-wrapper/build-model-image.sh

```diff
@@ -74,8 +74,37 @@ MODEL_ARCHIVE_FILES=$WORKDIR/$MODEL_DIR/archive.zip
 MODEL_VARIABLE_FILES="$(ls $WORKDIR/$MODEL_DIR/*.properties | xargs | sed 's/ /,/g')"
 CHOWN_ROOT="--chown oracle:root"
 
+if [[ ${MODEL_IMAGE_TAG} == *"-CM"* ]]; then
+cat << EOF
+dryrun:#!/bin/bash
+dryrun:# Use this script to build image '$MODEL_IMAGE_NAME:$MODEL_IMAGE_TAG'
+dryrun:# using the contents of '$WORKDIR/$MODEL_DIR'.
+dryrun:
+dryrun:set -eux
+dryrun:
+dryrun:rm -rf $WORKDIR/cm-image/${MODEL_IMAGE_TAG}
```

Comment:
This rm -rf is a bit scary IMO - the entire file system would be deleted if WORKDIR or MODEL_IMAGE_TAG were accidentally defined as `/ /tmp`. To at least partly defend, something like the following should hopefully limit the damage regardless of their value:
```
cd "$WORKDIR"
[ -d "cm-image/${MODEL_IMAGE_TAG}" ] && rm -rf "cm-image/${MODEL_IMAGE_TAG}"
```